magento/pwa-studio
# [PWA-839] Localize Search

## Description

Translate strings within the Search area

## Related Issue





https://jira.corp.magento.com/browse/PWA-839

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to Search areas of the application
2. Verify strings are translated where applicable.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
